docs/troubleshoot-collection.md

```diff
@@ -340,6 +341,31 @@ You [check Prometheus logs](#prometheus-logs) to verify there are no errors duri
 
 You can also check `prometheus_remote_storage_.*` metrics to look for success/failure attempts.
 
+### Missing metrics for ArgoCD installation
+
+There is known issue with Argo CD and metrics collection. If you override `spec.source.helm.releaseName`, then Kube State and Node metrics are not collected due to the following:
```

Comment:
```suggestion
There is known issue with Argo CD and metrics collection. If you override `spec.source.helm.releaseName` in the `ApplicationSet`,  which is used to configure your application in Argo CD, then Kube State and Node metrics are not collected due to the following:
```

Reply:
@kkujawa-sumo Thanks for this suggestion. I applied it with some change, because there are both `Application` and `ApplicationSet`. I have been testing on `Application` and customer reported issue for `ApplicationSet`